gef-ui/package.json

```
@@ -19,13 +19,14 @@
   "author": "UK Export Finance",
   "main": "index.js",
   "scripts": {
-    "api-test": "jest --coverage --verbose --forceExit --config=api-test.jest.config.js",
+    "api-test": "jest --verbose --forceExit --config=api-test.jest.config.js",
```

```suggestion
    "api-test": "jest --coverage --verbose --forceExit --config=api-test.jest.config.js",
```
I think we committed this by accident